imports/api/deployables/channels/methods.js

```diff
@@ -31,57 +29,83 @@ const NonEmptyString = Match.Where((x) => {
 });
 
 Meteor.methods({
-    updateChannel(orgId, appId, channelName){
+    async updateChannel(orgId, appId, channelName){
         requireOrgAccess(orgId);
         check( orgId, String );
         check( appId, String );
         check( channelName, String );
 
         logUserAction(Meteor.userId(), 'updateChannel', `Update channel ${orgId}:${appId}:${channelName}`);
 
-        Channels.update(
-            { 
+        let client = await getQueryClient();
+        return await client.mutate({
+            mutation: gql`
+              mutation EditChannel($org_id: String!, $uuid: String!, $name: String!) {
+                editChannel(org_id: $org_id, uuid: $uuid, name: $name) { 
+                    uuid
+                    name
+                    success
+                  }
+              }
+            `,
+            variables: {
                 'org_id': orgId,
-                uuid: appId
-            }, 
-            { 
-                $set: 
-                { 
-                    'name': channelName,
-                    'updated': new Date() 
-                } 
-            });
-        return true;
+                'uuid': appId,
+                'name': channelName
+            }
+        }).catch( (err) => {
+            throw new Meteor.Error(err.message);
```

Comment:
Is there a way we can somehow force the query/mutate funcs to push any errors into a new Meteor.Error() so we dont have to run this every time?